Update Readme with Batch Query Example (#26900)
Azure/azure-sdk-for-js: sdk/monitor/monitor-query/README.md

const logsQueryClient = new LogsQueryClient(credential);
// or
const metricsQueryClient = new MetricsQueryClient(credential);
// or
const batchEndPoint: string = "<YOUR_METRICS_ENDPOINT>"; //for example, https://eastus.metrics.monitor.azure.com/

> const metricsBatchQueryClient: MetricsBatchQueryClient = new MetricsBatchQueryClient(

Reviewer: Please add the types to the LogsQueryClient and MetricsQueryClient above, so that they match your syntax for MetricsBatchQueryClient

- [Metrics query](#metrics-query)
  - [Handle metrics query response](#handle-metrics-query-response)
  - [Example of handling response](#example-of-handling-response)

> - [Batch query](#batch-query)

Reviewer: ```suggestion
- [Batch metrics query](#batch-metrics-query)
```

A full sample can be found [here](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-query/samples/v1/typescript/src/metricsQuery.ts).

> ### Batch query

Reviewer: ```suggestion
### Batch metrics query
```

> The following example executes multiple logs queries in a single request using the `MetricsBatchQueryClient.queryBatch` method:

Reviewer: ```suggestion
The following example executes multiple metrics queries in a single request using the `MetricsBatchQueryClient.queryBatch` method:
```

```ts
let resourceIds: string[] = [
  "/subscriptions/0000000-0000-000-0000-000000/resourceGroups/test/providers/Microsoft.OperationalInsights/workspaces/test-logs",
  "/subscriptions/0000000-0000-000-0000-000000/resourceGroups/test/providers/Microsoft.OperationalInsights/workspaces/test-logs2",
];
let metricsNamespace: string = "<YOUR_METRICS_NAMESPACE>";
let metricNames: string[] = ["requests", "count"];

> const batchEndPoint: string = "<YOUR_METRICS_ENDPOINT>"; //Eg: https://eastus.metrics.monitor.azure.com/

Reviewer: ```suggestion
const batchEndPoint: string = "<YOUR_METRICS_ENDPOINT>"; //for example, https://eastus.metrics.monitor.azure.com/
```

> const clientSecretCredential = new ClientSecretCredential(<YOUR_TENANT_ID>, <YOUR_CLIENT_ID>, <YOUR_CLIENT_SECRET>);

Reviewer: Use DefaultAzureCredential instead
